### ISSUE TITLE
Ubuntu: Installing using password

### ISSUE THREAD

**Issue by ejohansson** [mysqludf/lib_mysqludf_preg#20]
Server:

```
Distributor ID: Ubuntu
Description:    Ubuntu 12.04.2 LTS
Release:    12.04
Codename:   precise
```

Installation procedure

``` sh
apt-get install libmysqlclient-dev libpcre3-dev
git clone https://github.com/mysqludf/lib_mysqludf_preg.git
cd lib_mysqludf_preg
./configure; make  install; make installdb ; make test
```

Error

```
...
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: NO)
make: *** [uninstalldb] Error 1
cd test; make test
make[1]: Entering directory `/root/mysqludf/lib_mysqludf_preg/test'
/usr/bin/mysqltest --defaults-file=/etc/mysql/debian.cnf --include=create_testdb.sql  --result-file=lib_mysqludf_preg_capture.result < lib_mysqludf_preg_capture.test
mysqltest: At line 19: query 'SELECT PREG_CAPTURE( '/(x)(?:(abc)|(xyz))(x)/' , 'xabcx' , 2 ) ' failed: 1305: FUNCTION preg_test.PREG_CAPTURE does not exist
not ok
make[1]: *** [lib_mysqludf_preg_capture.run] Error 1
make[1]: Leaving directory `/root/mysqludf/lib_mysqludf_preg/test'
make: *** [test] Error 2
...
```

**tl;dr;**
`Access denied for user 'root'@'localhost' (using password: NO)`

**How do I configure the installation to run using a password?**

**Comment by ejohansson** [comment]
Duplicate of #19 read [this comment](https://github.com/mysqludf/lib_mysqludf_preg/issues/19#issuecomment-101952155)

``` sh
apt-get install libmysqlclient-dev libpcre3-dev
apt-get install libtool automake autoconf autogen
git clone https://github.com/mysqludf/lib_mysqludf_preg.git
cd lib_mysqludf_preg
aclocal; automake --add-missing; ./configure; make; make install; make MYSQL="mysql -p[PASSWORD_HERE]" installdb; make test;
```
